webhook_receiver/settings/__init__.py

```diff
@@ -158,7 +157,10 @@
         'DJANGO_DATABASE_URL was set to an empty string. Provide a '
         'valid URL or unset DJANGO_DATABASE_URL to use the default.'
     )
-
+# If the database URL sets options via "?key=value" parameters, we
+# want to use them.  But if it doesn't, we still want to set OPTIONS
+# to an empty dictionary, which django-environ doesn't do by default.
+default_db.setdefault('OPTIONS', {})
```

Comment:
*This* is where I believe we should initialize `OPTIONS` (not in `settings.production`).

Reply:
I agree. This is better